drivers/hmis/config/initializers/hmis_feature.rb

```diff
@@ -23,3 +23,10 @@
   HmisDataCleanup::Util.fix_disabling_condition_nils!
   HmisDataCleanup::Util.fix_race_gender_99s!
 end
+
+TodoOrDie('Remove one-time job', by: '2025-10-01')
```

Comment:
Can we say 10/6 maybe? 10/1 is a loaded day with fy26 release so just thinking to avoid mental load of new errors